release_images.sh

```diff
@@ -14,10 +15,10 @@
 # limitations under the License.
 #
 
-FROM golang:1.12.7-alpine3.10
-RUN apk add --no-cache git
 
-COPY boilerplate /boilerplate
+user="fnproject"
+image="go"
 
-WORKDIR /boilerplate
-CMD ["/bin/sh", "mod.sh"]
+runtime115="1.15"
```

Comment:
There will be cases when we will need to release more versions than 1.15. So to keep it consistent can we modify this script to take version as an input similar to build script ?

Reply:
This is inherited from fdk-python's way of pushing images: https://github.com/fnproject/fdk-python/blob/master/release_images.sh

for more images, we add new entries to this and build-image shell scripts.